### ISSUE TITLE
Change the implicit account creation flow after the introduction of zero balance account

### ISSUE THREAD

**Issue by bowenwang1996** [near/NEPs#462]
Once https://github.com/near/NEPs/pull/448 is implemented, implicit accounts can be created without having to hold any balance, which means that they could be created through a `CreateAccountAction`. This means we don't have to special case the creation of implicit account (through transfers) and also reduce the cost of token transfers to implicit accounts as a result.

**Comment by ilblackdragon** [comment]
Curious if this change planned to be done? 

Because I'm not sure if this works given the expectation that someone (exchange or a wallet) sending funds to the account independently if it exists or not. This change would require all of them add additional checks now and an explicit account creation for implicit accounts (pun intended)

**Comment by bowenwang1996** [comment]
Yeah that is a good point. We currently have no plan of implementing it. This issue was more for keeping track of the idea as it came out of a discussion when we were implementing zero balance account
